emoncms9install

```diff
@@ -10,7 +10,7 @@
 # Install all dependencies
 # There are a few extra things in here such as mosquitto
 # which will become useful soon
-BRANCH="beaglebone"
```

Comment:
let's not make this 0.5.1 as else we will have to change this version . Please make it RELEASE